test/fixtures/gcs-fixture/src/main/java/fixture/gcs/GoogleCloudStorageHttpHandler.java

```diff
@@ -298,9 +298,10 @@ private String httpServerUrl(final HttpExchange exchange) {
                 skippedEmptyLine = markAndContinue && endPos == startPos;
                 startPos = endPos;
             } else {
-                // removes the trailing end "\r\n--__END_OF_PART__--\r\n" which is 23 bytes long
-                int len = fullRequestBody.length() - startPos - 23;
-                content = Tuple.tuple(name, fullRequestBody.slice(startPos, len));
+                while (isEndOfPart(fullRequestBody, endPos) == false) {
```

Comment:
The new SDK isn't just sending __END_OF_PART__${uuid} so this needed to become a little smarter.
See related SDK code https://github.com/googleapis/google-http-java-client/blob/master/google-http-client/src/main/java/com/google/api/client/http/MultipartContent.java#L35